storage/database/dynamodb_test.go

```diff
@@ -85,6 +90,59 @@ func (s *SuiteDynamoDB) TestDynamoDB_Put() {
 	s.NoError(returnedErr)
 }
 
+func (s *SuiteDynamoDB) TestDynamoDB_Timeout() {
```

Comment:
I am confused with this test. Could you add more comments for this test?
1) Instead of using the localstack, the code creates a new temp server `fakeEndpoint`?
2)Why are we using a new temp server instead of local stack?
3) Is there a purpose of setting the regions to `"ap-northeast-2"`? Why did you set different region for dynamoDB client and dynamoDB?

Reply:
I added more comments.
1. I wanna produce a network failure scenario. If the localstack is used, it is hard to drop dynamoDB requests allowing TCP handshake. 
2. I just used a test config already specified. The config is just used to create dynamoDB without any errors, so the actual values are not important. Anyway, I match up the region values as you remarked.